verifier/src/tcb.rs

```diff
@@ -123,6 +220,36 @@ impl TryFrom<&str> for Tcb {
     }
 }
 
+impl Tcb {
+    /// Returns true if this [`Tcb`] instance can represent the provided
+    /// `tcb_info`
+    ///
+    /// Performs the comparisons in 3.a and 3.b from
+    /// <https://api.portal.trustedservices.intel.com/documentation#pcs-tcb-info-v4>
+    ///
+    /// > 3. Go over the sorted collection of TCB Levels retrieved from TCB Info
+    /// >    starting from the first item on the list:
+    /// >   a. Compare all of the SGX TCB Comp SVNs retrieved from the SGX PCK
+    /// >      Certificate (from 01 to 16) with the corresponding values of SVNs
+    /// >      in sgxtcbcomponents array of TCB Level. If all SGX TCB Comp SVNs
+    /// >      in the certificate are greater or equal to the corresponding
+    /// >      values in TCB Level, go to 3.b, otherwise move to the next item
+    /// >      on TCB Levels list.
+    /// >   b. Compare PCESVN value retrieved from the SGX PCK certificate with
+    /// >      the corresponding value in the TCB Level. If it is greater or
+    /// >      equal to the value in TCB Level, read status assigned to this
+    /// >      TCB level. Otherwise, move to the next item on TCB Levels list.
+    fn is_corresponding_level(&self, tcb_info: &PckTcb) -> bool {
```

Comment:
Up for suggestions on this name.
The "level" feels off here, maybe this method should be brought up into the `TcbLevel` struct...??

Reply:
i'd suggest maybe a name like `matches_pck_tcb` ?

Reply:
I'm still chewing on this, will try to think overnight.

I may be confounding words again but `matches` makes me think `equals`, even `corresponding` kind of feels that way.
maybe `window`...
`is_tcb_level_window`?? IDK

Reply:
still undecided, punting on naming for now, it's internal to the module so renaming later shouldn't be too difficult